blog/content/post/2022-09-21-qubes-os-summit-2022.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Qubes OS summit 2022 - Summary
+abstract: 'Abstract first sentence.
+          Abstract second sentence.
+          Abstract third sentence.'
+cover: /covers/image-file.png
+author: norbert.kaminski
+layout: post
+published: true
+date: 2022-09-21
+archives: "2022"
+
+tags:
+  - Qubes
+  - Conference
+categories:
+  - Firmware
+  - IoT
+  - Miscellaneous
+  - OS Dev
+  - App Dev
+  - Security
+  - Manufacturing
+
+---
+
+## Long story short
+
+Three weeks ago 3mdeb with Qubes OS team had organized next edition of the
+Qubes OS summit. The last years [2020](https://blog.3mdeb.com/2020/2020-06-17-qubes_summation/)
+and [2021](#) editions were hosted
+remotely and this year event was a face-to-face event hosted in Berlin, which
+took place from the 9th to the 11th of September. It has been another productive
+event, during which we have exchanged loads of knowledge and expertise.
+
+## Day 1
+
+The first day started with
```

Comment:
I guess embedded yt video would be better. Description of talks could be more robust adding some personal review or some personal conclusion based on the talk.

Reply:
I added links to the video and I've changed a lot a descriptions of the presentations.